pkgs/development/python-modules/pgspecial/default.nix

```
@@ -1,4 +1,11 @@
-{ lib, buildPythonPackage, fetchPypi, pytest, psycopg2, click, sqlparse }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
```

```suggestion
, pytestCheckHook
```

and then instead of `checkPhase` set the following:

```
  preChecK = ''
    # can you check if these still exist?
    find tests -name \*.pyc -delete
  '';

  checkInputs = [
    configobj
    pytestCheckHook
  ];
```

---

There are no pyc files there during the check phase, and there shouldn't be anyway with Python >= 3.8 so I'm dropping that bit